### PR TITLE
studentOnly and instructorOnly signin flags

### LINKED ISSUE #285: Signin should have a studentOnly parameter
An error will be returned if the resulting human is not a student.

bonus points if a instructorOnly parameter is added as well.

This is blocking the mobile applications, so labelled as P1

### PR DESCRIPTION
**Pull request by seveibar**
fixes #285

### CODE REVIEW

File: server/auth/local/index.js [hunk 1]
```diff
@@ -2,11 +2,14 @@
 
 import express from 'express';
 import passport from 'passport';
+import User from "../../api/user/user.model";
 import {signToken} from '../auth.service';
 
 var router = express.Router();
 
 router.post('/', function(req, res, next) {
+  var studentOnly = req.query.studentOnly;
```
Comment (kmcnellis) [inline server/auth/local/index.js]: Its a post request, so should be using req.body.

**Comment by kmcnellis** [comment]
Please include repro steps and API docs next time.

**Comment by kmcnellis** [comment]
To verify the fix, add `instructorOnly: true` or `studentOnly : true` to the login request in  `client/app/account/login/login.controller.js`

**Comment by seveibar** [comment]
good modifications

**Pull request merged**